build.gradle

```diff
@@ -82,15 +82,19 @@ configure(subprojects.findAll {it.name.endsWith('.web')} ) {
         implementation 'org.springframework.boot:spring-boot-starter-hateoas'
         implementation 'org.springframework.boot:spring-boot-starter-web'
         implementation 'org.springframework.data:spring-data-commons'
-        // TODO Remove dependency on Spring Restdocs because the API documentation will be based on OpenAPI
         testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc'
     }
 }
 
 configure(subprojects.findAll {it.name.endsWith('.persistence')} ) {
     dependencies {
         implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
-        runtimeOnly 'com.h2database:h2'
+        // To avoid costs for unused CloudSQL instances it is possible to run the backend with H2
+        if (project.hasProperty('useH2')) {
```

Comment:
Use H2 by default for local tests from IDE, but Postgres by default in GitHub Actions